endpoints are allowed to retransmit entire payload of a lost packet (#2312)
quicwg/base-drafts: draft-ietf-quic-transport.md

priorities specified by the application indicate otherwise (see
{{stream-prioritization}}).

Even though a sender is encouraged to assemble frames containing up-to-date
information every time it sends a packet, it is not forbidden to retransmit

> frames of lost packets as they are.  A receiver MUST accept packets containing

Reviewer: ```suggestion
copies of frames from lost packets.  A receiver MUST accept packets containing
```
